app/Models/Event.php

```diff
@@ -88,19 +88,14 @@ public static function generate($type, $options)
 
             case 'beatmapsetApprove':
                 $beatmapset = $options['beatmapset'];
-
-                $beatmapsetUrl = e(route('beatmapsets.show', $beatmapset, false));
-                $beatmapsetTitle = e($beatmapset->artist.' - '.$beatmapset->title);
-                $userName = e($beatmapset->user->username);
-                $userUrl = e(route('users.show', $beatmapset->user, false));
+                $beatmapsetParams = static::beatmapsetParams($beatmapset);
+                $userParams = static::userParams($options['beatmapset']->user);
                 $approval = e($beatmapset->status());
 
-                $textCleanBeatmapsetUrl = $GLOBALS['cfg']['app']['url'].$beatmapsetUrl;
-                $textCleanUserUrl = $GLOBALS['cfg']['app']['url'].$userUrl;
-                $textClean = "[{$textCleanBeatmapsetUrl} {$beatmapsetTitle}] by [{$textCleanUserUrl} {$userName}] has just been {$approval}!";
+                $textClean = "[{$beatmapsetParams['url_clean']} {$beatmapsetParams['title']}] by [{$userParams['url_clean']} {$userParams['username']}] has just been {$approval}!";
```

Comment:
You mean like https://github.com/ppy/osu-web/pull/11741/commits/b2937ec62e178ae807a47484787f6038c9dbe75f?

I'd say it's a beneficial change yeah, because only thanks to it I noticed two of the events I added had quotes missing in the `text_clean` variant that should have been there :sweat_smile: